Source/WebInspectorUI/UserInterface/Views/OverrideUserPreferencesPopover.css

```diff
@@ -37,6 +37,10 @@
     margin-block-end: 4px;
 }
 
+.popover .user-preferences-content > h1:not(:first-of-type) {
```

Comment:
`h1:not(:first-of-type)` is probably fine, but it seems like what we really mean is `h1:not(:first-child)`, which should be faster to lay out for the engine since it doesn't rely on knowledge of the other children? Also makes sure that if we were to add a control without a header above later, the header below that control would still correctly have a block-start margin.

Reply:
That's clever! I hadn't considered the selector performance. Good idea to ensure a `block-start` margin applies if the DOM structure changes in the future.